### ISSUE TITLE
Missing label for `public` checkbox input

### ISSUE THREAD

**Issue by aviannamiller** [omeka/plugin-Ngram#31]
On "Add Corpus":

- `public` checkbox input is missing label due to an incorrect assignment (label is currently "for" `name`).
